lib/tweetflood/twitter_stream.ex

```diff
@@ -88,8 +88,9 @@ defmodule Tweetflood.TwitterStream do
   end
 
   @doc false
-  defp part_of_campaign?(tweet) do
-    Enum.all?(filter_words(), &String.contains?(tweet.full_text, &1))
+  defp part_of_campaign?(%{full_text: full_text}) do
+    lower_tweet = String.downcase(full_text)
+    Enum.any?(filter_words(), &String.contains?(lower_tweet, &1))
```

Comment:
@btkostner we were using `all?/2` before, was that intentional? I _think_ neither really matters given we only have a single hashtag we're looking for.

Reply:
I put `all?/2` just in case we wanted to check against two words or something. It's not _not needed_ but doesn't hurt.